app/components/AlbumView/index.js

```diff
@@ -202,65 +204,36 @@ class AlbumView extends React.Component {
           </tr>
         </thead>
         <tbody>
-          {album.tracklist.map((el, i) =>
-            this.renderContextPopup(album, el, i)
+          {album.tracklist.map((track, index) => {
+            track.artist = this.getArtistName(track, album);
+            track.name = track.title;
+            if (parseInt(track.duration) !== track.duration) {
+              track.duration = Utils.stringDurationToSeconds(track.duration);
+            }
+            _.set(track, 'image[0][#text]', _.get(album, 'images[0].uri'));
+
+            return (<TrackRow
+              key={'album-track-row-' + index}
+              track={track}
+              index={'album-track-' + index}
+              clearQueue={this.props.clearQueue}
+              addToQueue={this.props.addToQueue}
+              startPlayback={this.props.startPlayback}
+              selectSong={this.props.selectSong}
+              musicSources={this.props.musicSources}
+              displayCover={false}
+              displayTrackNumber={true}
```

Comment:
for boolean props that are set to true, you only need to put their name there (as in `displayTrackNumber` instead of `displayTrackNumber={true}`. Instead of setting props to false, you can simply omit them and they'll be false by default.